be/src/exec/vectorized/mysql_scan_node.cpp

```diff
@@ -82,7 +83,101 @@ Status MysqlScanNode::open(RuntimeState* state) {
     RETURN_IF_CANCELLED(state);
     SCOPED_TIMER(_runtime_profile->total_time_counter());
     RETURN_IF_ERROR(_mysql_scanner->open());
-    RETURN_IF_ERROR(_mysql_scanner->query(_table_name, _columns, _filters, _limit));
+
+    // Get [slot_id, slot] map
+    std::unordered_map<SlotId, SlotDescriptor*> slot_by_id;
+    for (SlotDescriptor* slot : _tuple_desc->slots()) {
+        slot_by_id[slot->id()] = slot;
+    }
+
+    std::unordered_map<std::string, std::list<std::string>> filters_in;
```

Comment:
Why not use slot id as key?

Reply:
> Why not use slot id as key?

Because we should collect [field_name, list_value], this is more convenient